contracts/DummyValidator.sol

```diff
@@ -1,18 +1,34 @@
 pragma solidity ^0.4.21;
 
 import { Jurisdiction } from "./tpl-contracts/Jurisdiction.sol";
+import { TransactionChecker } from "./tpl-contracts/checks/TransactionChecker.sol";
 
 
-contract DummyValidator {
+contract DummyValidator is TransactionChecker {
 
   Jurisdiction jurisdiction;
 
-  function DummyValidator(Jurisdiction _jurisdiction) public {
+  function DummyValidator(Jurisdiction _jurisdiction)
+    TransactionChecker(_jurisdiction)
+    public
+  {
     jurisdiction = _jurisdiction;
   }
 
   function validate() public {
     jurisdiction.addAttribute(msg.sender, "VALID", 1);
```

Comment:
I found this weird, because the `Registry` doesn't have `addAttribute`, just `hasAttribute`.
Maybe, the idea is to have a checker that just checks `hasAttribute`, and a validator that calls `addAttribute`.
Or maybe we should include `addAttribute` to `Registry`, so it's nicer to do both on the same validator.

Reply:
`Registry` seems to be the read-only interface of `Jurisdiction`. :man_shrugging: 

It looks good to me like this.

Reply:
What's weird is that now the validator has a variable called registry and another called jurisdiction, and they point to the same instance.

Reply:
Ah, right. I don't see a way out of this. We can't cast the `registry` variable safely because we don't know if `TransactionChecker` ever changes it to be something different.